ftw/http.py

```diff
@@ -433,8 +433,8 @@ def get_response(self):
                 err.errno == errno.WSAEWOULDBLOCK:
                         pass
                 elif (self.request_object.protocol == 'https' and
-                      sys.exc_info()[0].__name__ == 'SSLWantReadError'):
-                    pass
+                    err[0] == ssl.SSL_ERROR_WANT_READ):
```

Comment:
@zmallen the condition on 436 was never true. I am not sure how sys.exc_info()[0] gets set here, but it wasn't matching. As a result, the recv operation was failing because in was still in the process of waiting for, decrypting or verifying data. The failure wasn't permanent, it was returning ssl.ERROR_WANT_READ telling us it wanted a bit more time. So I adjusted the condition